spec/fixtures/application/1.0/application.json

```diff
@@ -125,13 +159,22 @@
           "payment_type": "child",
           "amount": 3990,
           "frequency": "month",
+          "ownership_type": "applicant",
           "metadata": {}
         },
         {
           "payment_type": "other",
           "amount": 1884,
           "frequency": "fortnight",
+          "ownership_type": "applicant",
           "metadata": { "details": "Top up" }
+        },
+        {
+          "payment_type": "working_or_child_tax_credit",
+          "amount": 1532,
+          "frequency": "month",
+          "ownership_type": "partner",
```

Comment:
I think these would either ALL be applicant, or ALL be applicant_and_partner. No mixture, and never Partner alone

Reply:
That's for `outgoings_payments`. For income payments and income benefits, it's either `applicant` or `partner`, never `applicant_and_partner`

Reply:
Duh, sorry!